source/js/foundation/pages/youtube-regrets/categories-bar-chart.js

```diff
@@ -86,9 +89,14 @@ export const initYouTubeRegretsCategoriesBarChart = () => {
           },
           ticks: {
             color: "white",
-            font: {
-              size: 12,
-              family: "Nunito Sans",
+            font: function (context) {
+              var width = context.chart.width;
+              const size = width >= 410 ? 12 : 8;
+
+              return {
+                family: "Nunito Sans",
+                size: size,
+              };
```

Comment:
Variable font sizing for the y-axis labels. It'll make the labels smaller so they aren't cut off by the viewport on small devices.